Another major refactor. When in ruby do as the rubyists do.

claco opened the pull request:
- put libraries files in the proper paths for their class names
- puts specs in paths matching files
- remove chefspec for old school rspec class tests
- break :join action down into smaller bits

claco committed:
Another major refactor. When in ruby do as the rubyists do.

- put libraries files in the proper paths for their class names
- puts specs in paths matching files
- remove chefspec for old school rspec class tests
- break :join action down into smaller bits

rcbjenkins commented:
Running GATE test(s): http://build.monkeypuppetlabs.com:8080/job/GIT-Pull-Request/474/console

rcbjenkins commented:
Gate: Submodule Foodcritic: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-foodcritic/437/consoleFull

rcbjenkins commented:
Gate: Submodule syntax: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-syntax/442/consoleFull

rcbjenkins commented:
Gate: Submodule Tailor (non-voting): SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-tailor/388/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-precise-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/632/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-centos-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/632/consoleFull

rcbjenkins commented:
Gate failed (mini-ha built using grizzly on jenkins-precise-v2).  See URL below for details.
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/632/consoleFull : FAILURE

rcbjenkins commented:
Gate failed (mini-ha built using grizzly on jenkins-centos-v2).  See URL below for details.
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/632/consoleFull : FAILURE

claco committed:
Fix resource loading error

Apparently, chefspec will load all files in all folders in libraries but
chef server proper does not. I dislike that Chef forces bad habits upon
me. :-/

rcbjenkins commented:
Running GATE test(s): http://build.monkeypuppetlabs.com:8080/job/GIT-Pull-Request/475/console

rcbjenkins commented:
Gate: Submodule Foodcritic: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-foodcritic/438/consoleFull

rcbjenkins commented:
Gate: Submodule syntax: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-syntax/443/consoleFull

rcbjenkins commented:
Gate: Submodule Tailor (non-voting): SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-tailor/389/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-precise-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/633/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-centos-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/633/consoleFull

rcbjenkins commented:
Gate failed (mini-ha built using grizzly on jenkins-centos-v2).  See URL below for details.
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/633/consoleFull : FAILURE

rcbjenkins commented:
Gate failed (mini-ha built using grizzly on jenkins-precise-v2).  See URL below for details.
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/633/consoleFull : FAILURE

rcbjenkins commented:
Running GATE test(s): http://build.monkeypuppetlabs.com:8080/job/GIT-Pull-Request/476/console

rcbjenkins commented:
Gate: Submodule Foodcritic: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-foodcritic/439/consoleFull

rcbjenkins commented:
Gate: Submodule syntax: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-syntax/444/consoleFull

rcbjenkins commented:
Gate: Submodule Tailor (non-voting): SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-tailor/390/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-precise-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/634/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-centos-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/634/consoleFull

rcbjenkins commented:
Gate failed (mini-ha built using grizzly on jenkins-centos-v2).  See URL below for details.
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/634/consoleFull : FAILURE

rcbjenkins commented:
Gate failed (mini-ha built using grizzly on jenkins-precise-v2).  See URL below for details.
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/634/consoleFull : FAILURE

rcbjenkins commented:
Running GATE test(s): http://build.monkeypuppetlabs.com:8080/job/GIT-Pull-Request/477/console

rcbjenkins commented:
Gate: Submodule Foodcritic: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-foodcritic/440/consoleFull

rcbjenkins commented:
Gate: Submodule syntax: SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-syntax/445/consoleFull

rcbjenkins commented:
Gate: Submodule Tailor (non-voting): SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-submodule-tailor/391/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-precise-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/635/consoleFull

rcbjenkins commented:
Starting Matrix Gate Job: (jenkins-centos-v2), console URL:  http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/635/consoleFull

rcbjenkins commented:
Gate:  Nova AIO (jenkins-centos-v2): SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-centos-v2,label=nova/635/consoleFull

rcbjenkins commented:
Gate:  Nova AIO (jenkins-precise-v2): SUCCESS
- http://build.monkeypuppetlabs.com:8080/job/gate-nova-matrix/BUILD_TYPE=mini-ha,INSTANCE_IMAGE=jenkins-precise-v2,label=nova/635/consoleFull
